mongo/integration/mtest/setup.go

```diff
@@ -179,9 +179,10 @@ func Setup(setupOpts ...*SetupOptions) error {
 		}
 	}
 
-	// For load balanced clusters, retrieve the required LB URIs and add additional information (e.g. TLS options) to
+	// For non-serverless, load balanced clusters, retrieve the required LB URIs and add additional information (e.g. TLS options) to
 	// them if necessary.
-	if testContext.topoKind == LoadBalanced {
+	testContext.serverless = os.Getenv("SERVERLESS") == "serverless"
+	if !testContext.serverless && testContext.topoKind == LoadBalanced {
```

Comment:
As @prestonvasquez pointed out, the serverless task was asserting that the `*_MONGOS_LB_URI` env variables were set in this block. They are no longer set for the serverless task, so I've updated this conditional and the one in `getURIForClient()`.